dcrutil/wif_test.go

```diff
@@ -193,7 +193,7 @@ func TestEncodeDecodeWIF(t *testing.T) {
 				t.Errorf("NewWIF failed: want '%v', got '%v'", test.wif, got)
 			}
 
-			w.SerializePubKey()
+			w.PubKey()
```

Comment:
your guess is as good as mine on this one.

Reply:
Yeah, just nuke this please.  I'd rather the test coverage show properly than trying to fake it like this.